deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -769,3 +768,17 @@ async def get_versions(request: Request):
     except Exception as e:
         logger.error(f"Failed to retrieve info of pulse3d versions: {repr(e)}")
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@app.get("/usage-quota", response_model=Union[UsageQuota, GenericErrorResponse])
```

Comment:
I would just call this `/usage`

Reply:
the `GenericErrorResponse` never gets returned here so I don't think it's necessary as a `response_model`

Reply:
and if this exists now, i don't know if it's necessary to check in the jobs and uploads routes

Reply:
I think this is still necessary because those are the checks that will prevent an analysis if a job gets through the FE for a user that has used all his quota

Reply:
Eventually, we'll want to remove it from the jobs and uploads routes and just call this route before posting a new job etc and only call POST /jobs POST /uploads if this new route returns an unreached quota. But for now to get this out, we can just keep it.